.drone.yml

```diff
@@ -28,7 +28,7 @@ build:
       - tests/wait_until_previous_builds_complete.sh
 
   vic:
-    image: golang:1.8
+    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.22}
```

Comment:
eminent-nation-87317 is the google cloud project id. AFAIK we can not change it.